sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateProperties.cs

```diff
@@ -8,60 +8,81 @@
 
 namespace Azure.Security.KeyVault.Certificates
 {
-    public class CertificateBase : IJsonDeserializable
+    public class CertificateProperties : IJsonDeserializable
     {
+        private const string IdPropertyName = "id";
+        private const string X509ThumprintPropertyName = "x5t";
+        private const string TagsPropertyName = "tags";
+        private const string AttributesPropertyName = "attributes";
+
         private CertificateAttributes _attributes;
 
         /// <summary>
-        /// The Id of the certificate
+        /// The Id of the certificate.
         /// </summary>
         public Uri Id { get; private set; }
 
         /// <summary>
-        /// The name of the certificate
+        /// The name of the certificate.
         /// </summary>
         public string Name { get; private set; }
 
         /// <summary>
-        /// The Uri of the vault in which the certificate is stored
+        /// The Uri of the vault in which the certificate is stored.
         /// </summary>
         public Uri VaultUri { get; private set; }
 
         /// <summary>
-        /// The version of the certificate
+        /// The version of the certificate.
         /// </summary>
         public string Version { get; private set; }
 
         /// <summary>
-        /// The digital thumbprint of the certificate which can be used to uniquely identify it
+        /// The digital thumbprint of the certificate which can be used to uniquely identify it.
         /// </summary>
         public byte[] X509Thumbprint { get; private set; }
 
         /// <summary>
-        /// The tags applied to the certificate
+        /// The tags applied to the certificate.
```

Comment:
The property below is null by default. Is that OK?

Reply:
Every other member except `Properties` is. I'm not aware of and couldn't find any .NET guidance on this in our docs. Should it not be null by default? I made sure `Properties` wouldn't be so it retains the same "feel" as before when we inherited properties, rather than a lazy initialization pattern like we do for `Tags`.

Reply:
@KrzysztofCwalina and @schaabs, I'll open a new issue to continue discussing this so I can unblock Scott who's waiting to build on this PR.